mkdocs.yml

```diff
@@ -0,0 +1,4 @@
+site_name: 'evergreen'
```

Comment:
What is this file for? Does it tell backstage where to find the Evergreen docs?

Reply:
Backstage is using [mkdocs](https://www.mkdocs.org/) to display the docs. This [minimal configuration](https://www.mkdocs.org/getting-started/) is telling mkdocs what to call the page and which files to ignore.